scripts/generate_evm_docs.py

```diff
@@ -0,0 +1,81 @@
+import os
+import re # Import regex module
+from jinja2 import Environment, FileSystemLoader # Re-add Jinja import
+
+# --- Configuration ---
+# Assumes the script is run from the workspace root
+TEMPLATE_DIR = "sources/_base_sources/evm"
+TEMPLATE_NAME = "_template.md.j2" # Use the markdown template
+OUTPUT_DIR = "sources/_base_sources/evm"
+OUTPUT_FILENAME_SUFFIX = "_docs_block.md" # Output markdown files
+# ---------------------
+
+# --- Remove the separate ETH_WITHDRAWALS_DOC constant ---
+# ETH_WITHDRAWALS_DOC = """ ... """
+
+def main():
+    print("Starting EVM documentation file generation...")
+
+    # Ensure output directory exists
+    if not os.path.exists(OUTPUT_DIR):
+        print(f"Creating output directory: {OUTPUT_DIR}")
+        os.makedirs(OUTPUT_DIR)
+
+    # --- Use Hardcoded Chain List --- (Same as YAML generation script)
+    chains = [
+        "abstract", "apechain", "arbitrum", "nova", "avalanche_c", "b3", "base",
+        "berachain", "blast", "bnb", "bob", "boba", "celo", "corn", "degen",
+        "ethereum", "fantom", "flare", "gnosis", "goerli", "ink", "kaia", "lens", "linea",
+        "mantle", "mode", "opbnb", "optimism", "polygon", "zkevm", "ronin",
+        "scroll", "sei", "shape", "sepolia", "sonic", "sophon", "unichain", "viction", # Added sepolia back
+        "worldchain", "zksync", "zora"
+    ]
```

Comment:
do you use hardcoded list here, because the macro list of evm chains doesn't auto trigger this script to run?
adding a chain forces this to run?

Reply:
I think I am not able to read the macro for some reason and didn't give this one too much thought. 

Using this going forward, it's probably best to simply input one chain in here and generate the files for that instead of regening all the chains all the time for no good reason.